azkaban-common/src/main/java/azkaban/event/EventData.java

```diff
@@ -0,0 +1,34 @@
+package azkaban.event;
+
+import azkaban.executor.Status;
+
+/**
+ * Carries a snapshot of the status data, suitable for asynchronous message passing.
+ */
+public class EventData {
+
+  private Status status;
+  private String nestedId;
```

Comment:
I often see **nestedId** from fetchExecutions API. 

 "nodes" : [ {
    "nestedId" : "***",
    "in" : [ "***_a" ],
    "startTime" : 1474698890218,
    "updateTime" : 1474698890246,
    "id" : "***_b",
    "endTime" : 1474698890242,
    "type" : "command",
    "attempt" : 0,
    "status" : "SUCCEEDED"
  }, {
    "nestedId" : "****_a",
    "startTime" : 1474698890151,
    "updateTime" : 1474698890219,
    "id" : "****_a",
    "endTime" : 1474698890188,
    "type" : "command",
    "attempt" : 0,
    "status" : "SUCCEEDED"
}]